custom_components/frigate/update.py

```diff
@@ -0,0 +1,102 @@
+"""Update platform for frigate."""
+from __future__ import annotations
+
+import logging
+
+from homeassistant.components.update import UpdateEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ATTR_MODEL, CONF_URL
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import FrigateDataUpdateCoordinator, get_frigate_device_identifier
+from .const import ATTR_COORDINATOR, DOMAIN, NAME
+
+_LOGGER: logging.Logger = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Sensor entry setup."""
+    coordinator = hass.data[DOMAIN][entry.entry_id][ATTR_COORDINATOR]
+
+    entities = []
+    entities.append(FrigateContainerUpdate(coordinator, entry))
+    async_add_entities(entities)
+
+
+class FrigateContainerUpdate(UpdateEntity, CoordinatorEntity):  # type: ignore[misc]
+    """Frigate container update."""
+
+    _attr_title = "Frigate Container"
```

Comment:
Is 'container' the right word? Maybe 'FrigateServerUpdate' ? (e.g. if I run Frigate outside a container, theoretically I'd still expect to be able to use this and the naming would then be confusing).

Reply:
I think `Frigate Server`. Based on how it is in the UI (see screenshot) it doesn't make sense to me to put update in the name since it is a status and shows all the time.

<img width="318" alt="Screen Shot 2022-04-20 at 9 10 25 PM" src="https://user-images.githubusercontent.com/14866235/164364119-9390a145-1fb0-4a9e-9a1e-7e18bafd5f88.png">

Reply:
Works for me.